src/parser.js

```diff
@@ -0,0 +1,40 @@
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List';
+const { JSDOM } = jsdom;
+const nameTest = /^name/i;
+const linkTest = /^#/;
+let lines = '';
+let feed = '';
+let feedList = '';
+let tempLine = '';
+
+/**
+* Get feed data from url
+* process it to remove unnecessary information like "name" and square brackets
+*/
+request(url)
+  .then((data) => {
+    const dom = new JSDOM(data);
+    feedList = dom.window.document.querySelector('pre').textContent;
+    lines = feedList.split(/\r\n|\r|\n/);
+
+    lines.forEach((element) => {
+      if (!linkTest.test(element)) {
+        if (element.startsWith('[')) {
+          tempLine = element.replace(/\[href="|\[/, '');
+          tempLine = tempLine.replace(']', '');
+          feed += `${tempLine}\n`;
+        }
+        if (nameTest.test(element)) {
+          tempLine = element.replace(/^name = |^name= |^name=/i, '');
+          feed += `${tempLine}\n`;
+        }
+      }
+    });
+    // eslint-disable-next-line no-console
+    console.log(feed);
```

Comment:
Let's make this into a set of tests.  If you don't want to do them here, you can get someone else to work on it in a new PR.

Reply:
I'l remove that and write all the data into a file so Nick can use it for his issue

Reply:
No, don't do anything in files.  We will use Redis/Bull as our message bus, not the filesystem.  For now, just get this exposed as a set of functions, and next week we'll hook it all together.